src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,76 @@
+const Task = require('../../models/task/task');
+
+getAllTasks = (req, res) => {
+  try { 
+    Task.find().sort({ isCheck: 1 }).sort({ creationTime: -1 }).then(result => {
+      res.send({ data: result });
+    })
+  }
+  catch {
+    res.status(400).send("cant get all tasks");
+  }
+};
+
+createNewTask = (req, res) => {
+  req.body.isCheck = false;
```

Comment:
you still don't know what value isCheck has. You need to check it out. And here just do the destructuring:
`const { text, isCheck } = req.body;`

Reply:
and then check isCheck. If you didn't get false, then throw an error, for example with "throw new Error();". Otherwise, the code below is executed